client/client.go

```diff
@@ -251,7 +250,7 @@ func setParamsWithAuthorization(p *AuxiliaryParams, w *WrapperV2, params paramSe
 	setParams(p, w, params)
 }
 
-// CreateAuthToken creates an auth token using the latest client.
+// CreateAuthToken creates an auth token using the V2 client.
 func (w *WrapperV2) CreateAuthToken(email, password string, p *AuxiliaryParams) (*auth_tokens.CreateAuthTokenOK, error) {
 	params := auth_tokens.NewCreateAuthTokenParams().WithBody(&models.V4CreateAuthTokenRequest{
```

Comment:
Unrelated but `auth_tokens`?

Reply:
What's the question? The sub-package `auth_tokens` is generated (part of gsclientgen) based on api-spec tags.

Reply:
LoL